spiketools/plts/trials.py

```diff
@@ -119,3 +119,30 @@ def plot_rate_by_time(x_vals, y_vals, average=None, shade=None, labels=None,
 
     if stats:
         _add_significance(stats, sig_level=sig_level, ax=ax)
+
+
+def create_trial_title(label, avg_pre, avg_post, t_val=None, p_val=None):
```

Comment:
For the naming here, how about changing it to something like `create_raster_title`? Just to be consistent with the `create_heat_title` above.

Reply:
yeh, I like that! updated!